releases/v2.1.4.md

```diff
@@ -0,0 +1,102 @@
+---
+title: What&#39;s New in v2.1.4
+toc: true
+summary: Additions and changes in CockroachDB version v2.1.4 since version v2.1.3
+---
+
+## January 22, 2019
+
+Get future release notes emailed to you:
+
+<div class="hubspot-install-form install-form-1 clearfix">
+    <script>
+        hbspt.forms.create({
+            css: '',
+            cssClass: 'install-form',
+            portalId: '1753393',
+            formId: '39686297-81d2-45e7-a73f-55a596a8d5ff',
+            formInstanceId: 1,
+            target: '.install-form-1'
+        });
+    </script>
+</div>
+
+### Downloads
+
+<div id="os-tabs" class="clearfix">
+    <a href="https://binaries.cockroachdb.com/cockroach-v2.1.4.darwin-10.9-amd64.tgz"><button id="mac" data-eventcategory="mac-binary-release-notes">Mac</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v2.1.4.linux-amd64.tgz"><button id="linux" data-eventcategory="linux-binary-release-notes">Linux</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v2.1.4.windows-6.2-amd64.zip"><button id="windows" data-eventcategory="windows-binary-release-notes">Windows</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v2.1.4.src.tgz"><button id="source" data-eventcategory="source-release-notes">Source</button></a>
+</div>
+
+### Docker image
+
+{% include copy-clipboard.html %}
+~~~shell
+$ docker pull cockroachdb/cockroach:v2.1.4
+~~~
+
+### Enterprise edition changes
+
+- It is now possible to use AWS S3 temporary credentials for [`BACKUP`](../v2.1/backup.html)/[`RESTORE`](../v2.1/restore.html) and [`IMPORT`](../v2.1/import.html)/[`EXPORT`](../v2.1/export.html) using the `AWS_SESSION_TOKEN` parameter in the URL. [#33046][#33046] {% comment %}doc{% endcomment %}}
+
+### SQL language changes
+
+- Added support for the `pg_catalog` introspection table `pg_am` for both PostgreSQL versions 9.5 and 9.6, which changed the table significantly. [#33276][#33276]
+- Previously, CockroachDB did not consider the value of the right operand for `<<` and `>>` operators, resulting in potentially very large results and excessive RAM consumption. This has been fixed to restrict the range of these values to that supported for the left operand. [#33247][#33247]
+- Attempts to use some PostgreSQL built-in functions that are not yet supported in CockroachDB now cause a clearer error message, and also become reported in [diagnostics reporting](../v2.2/diagnostics-reporting.html), if diagnostics reporting is enabled, so as to gauge demand. [#33427][#33427] {% comment %}doc{% endcomment %}}
```

Comment:
This is for 2.1 but some of these links are going to 2.2––intentional?

Reply:
Not intentional. Fixed. Thanks.